.gitignore

```diff
@@ -3,3 +3,4 @@ tmp/
 /npm-debug.log
 .directory
 .kateproject
+.idea/
```

Comment:
This should live in your [global .gitignore][1].


[1]: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore